tests/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Source.Tests/DiagnosticAnalysisRunnerTests.cs

```diff
@@ -0,0 +1,140 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Autofac.Extras.Moq;
+using AutoFixture;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.MSBuild;
+using Microsoft.CodeAnalysis.Text;
+using Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers;
+using Microsoft.Extensions.Logging;
+using Moq;
+using Xunit;
+
+namespace Microsoft.DotNet.UpgradeAssistant.Steps.Source.Tests
+{
+    public class DiagnosticAnalysisRunnerTests
+    {
+        private readonly Fixture _fixture;
+        internal const string TestProjectPath = @"assets\TestProject.csproj";
+
+        private static IEnumerable<AdditionalText> AdditionalTexts
```

Comment:
Why this? Can you just use `Enumerable.Empty<AdditionalText>()` where necessary?